frappe/public/js/frappe/ui/keyboard.js

```diff
@@ -347,3 +348,20 @@ function close_grid_and_dialog() {
 		return false;
 	}
 }
+
+frappe.ui.keys.add_shortcut({
+	shortcut: "shift+t",
+	action: function (e) {
+		if (!frappe.user.has_role("System Manager")) {
+			return;
+		}
```

Comment:
I think I'd even restrict this (and the old System Console) to the Administrator user. In practice, a lot of non-technical, C-level people and C-level assistants have the "System Manager" role and I feel uneasy with the thought of them discovering this by accident an running vibe-coded commands. 😅